frontend/src/app/modules/auth/register-form/register-form.component.scss

```diff
@@ -56,6 +56,9 @@
         }
         button:disabled {
             cursor: default;
+            background: #e0d8d7;
+            color: #989898;
+            border: 1px solid #e0d8d7 !important;
```

Comment:
Can't we deal without !important?

Reply:
i will try find another way

> Please, fix github actions comments, I know it's not your changes



> Please, fix github actions comments, I know it's not your changes
Ok

Reply:
> Can't we deal without !important?

I cheched, it's work without !important